.github/workflows/ci.yml

```diff
@@ -5,10 +5,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest]
+        os: [macos-latest, windows-latest, ubuntu-20.04]
```

Comment:
What does pinning only Ubuntu do?

Reply:
To allow me to run CI on an old version of Python 3.8. It is temporary.